pmix_governance.tex

```diff
@@ -234,6 +231,22 @@ \subsubsection{ASC Meetings}
 Standard up for consideration during that meeting in the published
 agenda.
 
+ASC quarterly meeting agenda items must be submitted to the Co-Chairs
+at least one month in advance of the meeting. One month in advance of
+the meeting the Co-Chairs will announce the tentative agenda for review.
+Any proposal wishing to be considered for this meeting must be put forward
+(by assigning the proper label and notifying the Co-Chairs) before that time.

```

Comment:
Does "by assigning the proper label" refer to labels on Github?

Reply:
Yes. I can clarify that.

Reply:
Actually, re-reading the paragraph it is a bit repetitive here. I don't know if that sentence is adding anything. The notion of adding labels to the PR is described elsewhere. I may just strike that sentence.

Reply:
pushed on a new commit.